build/yaml/functionalTests-CI.yml

```diff
@@ -0,0 +1,26 @@
+# This runs under BotBuilder-Js-CI-yaml. Replaces classic build BotBuilder-JS-master-CI-node12.
```

Comment:
This comment doesn't really describe what this file does, provide proper description

Reply:
updated with new comment